optimum/onnxruntime/modeling_ort.py

```diff
@@ -632,21 +635,32 @@ def _output_shape_inference(self, axis_name: Union[str, int], dimensions: Dict[s
         """
         if isinstance(axis_name, int):
             return axis_name
-        result = dimensions.get(axis_name, None)
-        if result is None:
-            subparts = axis_name.split("+")
-            result = 0
-            for subname in subparts:
-                subname = subname.strip()
-                subresult = dimensions.get(subname, None)
-                if subresult is None:
-                    try:
-                        subresult = int(subname)
-                    except ValueError:
-                        result = None
-                        break
-                result += subresult
-        return axis_name if result is None else result
+        # It is actually covered below, but this is to make things faster.
+        elif axis_name in dimensions:
+            return dimensions[axis_name]
+
+        tokens = []
```

Comment:
Reading the code, it is not clear to me what this contains, at first sight. I understand it should be a list of int as strings, from the values of `dimensions`. Is this correct?

Reply:
It contains each subpart of `axis_name`, where each sub axis name as be changed to actual value using `dimensions`.